src/tests/SharpRaven.UnitTests/Utilities/HttpSimulator.cs

```diff
@@ -829,5 +830,10 @@ public void RemoveAt(int index)
         }
 
         #endregion
+
+        public void SetUser(string username)
```

Comment:
I would prefer to have this as an extension method, if possible, since the `HttpSimulator` class is copied from god knows where. I can't remember. :smiley: